python/lsst/ip/diffim/subtractImages.py

```diff
@@ -831,8 +831,6 @@ def _prepareInputs(self, template, science, visitSummary=None):
             self.log.info("Science variance scaling factor: %.2f", sciVarFactor)
             self.metadata.add("scaleScienceVarianceFactor", sciVarFactor)
 
-        # Erase existing detection mask planes.
-        #  We don't want the detection mask from the science image
```

Comment:
Thanks for removing this: it was always going to be misleading compared with the docstring of `updateMasks`.